<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControleST</class>
 <widget class="QWidget" name="ControleST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="STWidget" name="stWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelS">
      <property name="text">
       <string>s =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinS">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelT">
      <property name="text">
       <string>t =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinT">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>STWidget</class>
   <extends>QWidget</extends>
   <header>stwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinS</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ControleST</receiver>
   <slot>valorSpinSAlterado(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinT</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ControleST</receiver>
   <slot>valorSpinTAlterado(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valorSpinSAlterado(double)</slot>
  <slot>valorSliderSAlterado(int)</slot>
  <slot>valorSpinTAlterado(double)</slot>
  <slot>valorSliderTAlterado(int)</slot>
 </slots>
</ui>
